apps/tangle-dapp/app/claim/EligibleSection.tsx

```diff
@@ -206,13 +208,13 @@ const EligibleSection: FC<Props> = ({
           />
         </div>
 
-        <div className="flex flex-col gap-4 p-4 border rounded-xl border-mono-0 dark:border-mono-180 shadow-[0px_4px_8px_0px_rgba(0,0,0,0.08)] bg-glass dark:bg-glass_dark">
+        <div className="space-y-6 p-4 border rounded-xl border-mono-0 dark:border-mono-180 shadow-[0px_4px_8px_0px_rgba(0,0,0,0.08)] bg-glass dark:bg-glass_dark">
           <Typography variant="body1" fw="bold" ta="center">
-            You will receive the liquid balance of...
+            You will receive the total liquid balance of...
```

Comment:
remove 'liquid'. 

(the term [liquid](https://docs.tangle.tools/docs/tokenomics/allocation/#initial-liquid-tokens) can be confused as the initial free balance)